app/forms/waste_exemptions_engine/applicant_email_form.rb

```diff
@@ -13,9 +13,7 @@ def submit(params)
       # Assign the params for validation and pass them to the BaseForm method for updating
       self.confirmed_email = params[:confirmed_email]
 
-      attributes = { applicant_email: params[:applicant_email] }
-
-      super(attributes)
+      super(params.permit(:applicant_email))
```

Comment:
This will make mass-assignment to only operate on the `:applicant_email`, as the `permit` will override the previous `permit` which included `:confirmed_email`